bot/plotting/random_plot_generator.py

```diff
@@ -95,37 +96,55 @@ def random_plot_generator(
                     "SEI": sei,
                 })
 
-            models = [
-                pybamm.lithium_ion.DFN(
-                    options=options
-                ),
-                pybamm.lithium_ion.SPM(
-                    options=options
-                ),
-                pybamm.lithium_ion.SPMe(
-                    options=options
-                ),
+            solvers = [
+                pybamm.CasadiSolver(mode="safe"),
+                pybamm.CasadiSolver(mode="fast with events")
             ]
 
-            model = random.choice(models)
+            solver = random.choice(solvers)
```

Comment:
Why do you choose the solver at random?

Reply:
No particular reason, just to experiment with the solvers. Should I just use the "safe" mode?

Reply:
Technically they are both the same solver, with the difference that one can deal with events (safe) and the other one can't (fast). In general, you want to you fast when you don't have events and safe otherwise so typically it is not a choice. You could try other DAE solvers though if you want to compare them (e.g. Scipy DAEs I think?)

Reply:
That sounds interesting, I'll try setting it up on my windows machine. For now, should I just use the "safe" mode?

Reply:
Should I keep the "safe" mode for now? The "fast with events" mode sometimes results in the complete freezing of a simulation.

Reply:
sounds good